app/inputexpressionfunctions.cpp

```diff
@@ -38,38 +40,47 @@ QVariant ReadExifImgDirection::func( const QVariantList &values, const QgsExpres
     return QVariant();
 
   return QVariant( resultDouble );
+#elif defined( Q_OS_IOS )
+  QString  result = IOSImagePicker::readExif( filepath, GPS_DIRECTION_TAG );
```

Comment:
why it is called getExitInfo on android and `readExif `  on iOS? also it should be part of `IosUtils` so it has some consistency